drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -8,12 +8,19 @@
 
 module Types
   class HmisSchema::CeReferral < Types::BaseObject
+    # object is a Hmis::Ce::Referral
     field :id, ID, null: false
     field :opportunity, HmisSchema::CeOpportunity, null: false
     field :steps, [HmisSchema::CeReferralStep], null: false
     field :status, HmisSchema::Enums::CeReferralStatus, null: false
     field :client_id, ID, null: false
     field :client, Types::HmisSchema::Client, null: true
+    field :date_started, GraphQL::Types::ISO8601Date, null: false, method: :created_at
```

Comment:
This seems like something we can resolve as a DateTime, since we have the time. That will allow recently-created referrals to show as "3 hours ago" instead of "today" in the relative date display. Or if we decide to only show the date in the frontend, that's fine -- but lets resolve the full datetime so that we have that flexibility as the frontend design evolves.

Reply:
And maybe we should call it `created_at` or `date_created`? I'm on the fence there. There is a difference between the datetime when the referral was created, and the datetime when the first referral _step_ was initiated.

I'm also thinking about potential future scenarios where the top prioritized eligible client gets auto-referred. In that scenario, `created_at` feels more appropriate than `date_started` since the creation was system-initiated. (Date Started feels user-initiated to me somehow, maybe I'm overthinking things).

Reply:
Sounds good! I was relying too much on the WIP mocks, which have a "started" column. But you're right, thinking about it more, it seems preferable to keep the naming of this field consistent in the schema and database, unless there is a strong need for them to diverge.